src/integrationTest/resources/features/delete_logon_audit.feature

```diff
@@ -2,9 +2,9 @@ Feature: The application's DELETE logon endpoint
 
   Scenario: The backend is able to process logon DELETE requests
     Given LAU IdAm backend application is healthy
-    When I request POST "/audit/logon" logon endpoint using userId "77777"
-    And I request DELETE "/audit/logon/deleteAuditLogonRecord" logon endpoint
-    And I GET "/audit/logon" logon using userId "77777"
+    When I request POST "/audit/logon/" logon endpoint using userId "77777"
```

Comment:
When you remove the slash from the path in the controller this change will be redundant.

Reply:
Please remove the slash because this change will most probably break the front-end.